src/v/pandaproxy/schema_registry/validation.cc

```diff
@@ -451,7 +451,8 @@ std::optional<schema_id_validator> maybe_make_schema_id_validator(
   const std::unique_ptr<api>& api,
   const model::topic& topic,
   const cluster::topic_properties& props) {
-    auto mode = config::shard_local_cfg().enable_schema_id_validation();
+    auto mode = from_config(
+      config::shard_local_cfg().enable_schema_id_validation());
```

Comment:
there are plenty of places in the tree that interrogate the config:: enable_schema_id_validation value, but this is the only place where config/external representation is converted into internal sub-system representation.